.github/workflows/test.yml

```diff
@@ -168,7 +170,7 @@ jobs:
         -DCMAKE_BUILD_TYPE=${{ matrix.cmake_build_type }}
         -DBUILD_TESTING=YES
         -DLLVM_DIR=${{ github.workspace }}/${{ env.llvm_package_basename }}/lib/cmake/llvm
-        ${{ runner.os == 'macOS' && '-D CMAKE_Swift_COMPILER=swiftc' || '' }}
+        ${{ matrix.os == 'macos-13' && '-DCMAKE_Swift_COMPILER=swiftc -DCMAKE_OSX_SYSROOT=$(xcrun --show-sdk-path)' || '' }}
```

Comment:
The use `runner` was correct and intentional.  We don't care which version of macOS is being used and anyway, the matrix is a per-project configuration thing.

Reply:
Oh true, I didn't catch this when pasting something from the other project. I'll fix this.

Reply:
If by the other project you mean Hylo, we probably ought to make that one match this one.  I wish there was a simpler way to share GH actions configuration.

Reply:
If you're feeling ambitious, explore [this](https://docs.github.com/en/actions/sharing-automations/sharing-actions-and-workflows-with-your-organization)